Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+
+
+function HeaderTitle() {
+    return (
+        <div className="headTitle">
```

Comment:
Since *h2* and *p* are block level elements, they will each take up their own line. One option is to pull all text inside an *h2* element, then use inline, *span* elements to style the text.
`<h2>Lambda School <span className="thin-small-text">@LambdaSchool 29 may</span>